src/order/filter/basic.lean

```diff
@@ -380,6 +380,19 @@ have ∅ ∈ f ⊓ principal (- s), from h.symm ▸ mem_bot_sets,
 let ⟨s₁, hs₁, s₂, (hs₂ : -s ⊆ s₂), (hs : s₁ ∩ s₂ ⊆ ∅)⟩ := this in
 by filter_upwards [hs₁] assume a ha, classical.by_contradiction $ assume ha', hs ⟨ha, hs₂ ha'⟩
 
+lemma inf_neq_bot_iff {f g : filter α} :
```

Comment:
I think the convention is to use `ne` instead of `neq`.

Reply:
Thanks